config_defaults.yml

```diff
@@ -49,10 +49,14 @@
     - '/.yardopts'
     - '/spec/'
 .travis.yml:
+  stages:
+    - static
+    - spec
+    - beaker
```

Comment:
I like the idea of stages, I don't like beaker being enabled by default. This will place an unnecessary coupling of all newly generated PDK modules with beaker. Can you remove the beaker defaults and those who want to enable a beaker stage and add it via `.sync.yml`.

Reply:
This won't enabled beaker by default. The above is just [specifying the order of stages](https://docs.travis-ci.com/user/build-stages/#specifying-stage-order-and-conditions).

To [define a build stage](https://docs.travis-ci.com/user/build-stages/#how-to-define-build-stages) there must be jobs bound to the particular stage. As long `docker_sets` stays empty, no build jobs will be created for the beaker stage. Without beaker jobs, there will be nothing depending on gems listed in `:system_tests` group of Gemfile, as `bundler_args: --without system_tests` is set as default. This is overwritten within `docker_defaults` only, which templates all the beaker jobs.

I would be unhappy removing beaker from the default stages array, because you would need to overwrite the whole `stages` key in  your `.sync.yml` to get beaker inserted at the right place. The main goal should be to reduce the inhibition threshold for making beaker tests in modules.

Reply:
I think the point here has been cleared, so I resolve this one.